src/app/components/Modals/UnbondModal.tsx

```diff
@@ -18,6 +22,17 @@ export const UnbondModal = ({
   onSubmit,
   processing,
 }: UnbondModalProps) => {
+  const { data: networkInfo } = useNetworkInfo();
+  const unbondingTime = blocksToDisplayTime(
+    networkInfo?.params.bbnStakingParams?.latestParam?.unbondingTime,
+  );
+  const unbondingFeeBtc = maxDecimals(
+    satoshiToBtc(
+      networkInfo?.params.bbnStakingParams?.latestParam?.unbondingFeeSat ?? 0,
+    ),
+    8,
+  );
```

Comment:
This is not right place for this logic. Pass it via component's props, encapsulate into related service